native/cocos/bindings/manual/jsb_conversions.h

```diff
@@ -1230,8 +1230,6 @@ inline bool nativevalue_to_se(const ccstd::vector<T> &from, se::Value &to, se::O
         if constexpr (!std::is_pointer<T>::value && is_jsb_object_v<T>) {
             auto *pFrom = ccnew T(from[i]);
             nativevalue_to_se(pFrom, tmp, ctx);
-            tmp.toObject()->clearPrivateData();
-            tmp.toObject()->setPrivateData(pFrom);
```

Comment:
Do these two lines cause any issues?

Reply:
Not sure, there should be a problem before